biblib/tests/unit_tests/test_pagination.py

```diff
@@ -0,0 +1,123 @@
+"""
+Tests Views of the application
+"""
+
+import unittest
+import uuid
+from flask import url_for
+from biblib.models import db, User, Library, Permissions, MutableDict
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm.exc import NoResultFound
+from biblib.views import UserView, LibraryView, DocumentView, PermissionView, \
+    BaseView, TransferView, ClassicView
+from biblib.views import DEFAULT_LIBRARY_DESCRIPTION
+from biblib.tests.stubdata.stub_data import UserShop, LibraryShop
+from biblib.utils import get_item
+from biblib.biblib_exceptions import BackendIntegrityError, PermissionDeniedError
+from biblib.tests.base import TestCaseDatabase, MockEmailService, \
+    MockSolrBigqueryService
+
+
+
+
+class TestLibraryViews(TestCaseDatabase):
+    """
+    Base class to test the Library view for GET
+    """
+
+    def __init__(self, *args, **kwargs):
+        """
+        Constructor of the class
+
+        :param args: to pass on to the super class
+        :param kwargs: to pass on to the super class
+
+        :return: no return
+        """
+
+        super(TestLibraryViews, self).__init__(*args, **kwargs)
+        self.user_view = UserView
+        self.library_view = LibraryView
+
+        self.stub_user = self.stub_user_1 = UserShop()
+        self.stub_user_2 = UserShop()
+
+        self.stub_library = LibraryShop()
+
+    def test_library_pagination_default(self):
+        """
+        """
+
+        # Ensure a user exists
+        user = User(absolute_uid=self.stub_user.absolute_uid)
```

Comment:
This makes a user with a randomly generated ID. So there is no guarantee the user will have an ID = 10.